sdk/evaluation/platform-matrix.json

```
@@ -35,6 +35,19 @@
           "TestSamples": "false"
         }
       }
+    },
+    {
+      "Config": {
+        "Ubuntu2004_310": {
+          "OSVmImage": "env:LINUXVMIMAGE",
+          "Pool": "env:LINUXPOOL",
+          "PythonVersion": "3.10",
+          "CoverageArg": "--disablecov",
+          "TestSamples": "false",
+          "InjectedPackages": "pyrit==0.8.1",
+          "UnsupportedToxEnvironments": "mindependency,sdist"
```

To avoid running into _any_ unexpected `nightly` issues. Please update this `UnsupportedToxEnvironments` list to match the total list of our `nightly` tox envs EXCEPT for `whl`.

```suggestion
          "UnsupportedToxEnvironments": "sdist,depends,latestdependency,mindependency,whl_no_aio"
```